src/ifs/wl_seat/wl_pointer.rs

```diff
@@ -74,6 +74,7 @@ pub struct WlPointer {
     pub seat: Rc<WlSeat>,
     pub tracker: Tracker<Self>,
     last_motion: Cell<(Fixed, Fixed)>,
+    pub v120_accumulator: [Cell<i32>; 2],
```

Comment:
I think this should be reset to 0 in send_enter.

Reply:
I think axis_stop is only used for touchpads to signal that you've lifted the fingers from the pad. I think it makes sense to reset it there as well.

Reply:
pushed!